library/ui/src/main/java/com/google/android/exoplayer2/ui/PlayerControlView.java

```diff
@@ -196,6 +197,9 @@
   public static final int MAX_WINDOWS_FOR_MULTI_WINDOW_TIME_BAR = 100;
 
   private static final long MAX_POSITION_FOR_SEEK_TO_PREVIOUS = 3000;
+  private static final int DEFAULT_UPDATE_DP = 5;
```

Comment:
Just curious: How did you choose this value?

Reply:
By checking a couple of values, how it looks visually. 20dp was too high and the progress jumped way too much. In my opinion of course.